lib/ssz_types/beacon_block_header.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SszTypes.BeaconBlockHeader do
+  @moduledoc """
+  Struct definition for `BeaconBlockHeader`.
+  Related definitions in `native/ssz_nif/src/types/`.
+  """
+
+  fields = [
+    :slot,
+    :proposer_index,
+    :parent_root,
+    :state_root,
+    :body_root
+  ]
+
+  @enforce_keys fields
+  defstruct fields
+
+  @type t :: %__MODULE__{
+          slot: SszTypes.slot(),
+          proposer_index: SszTypes.proposer_index(),
```

Comment:
The type used in the spec is `validator_index`

Reply:
sure, I've changed it.